docker.py

```diff
@@ -9,12 +9,18 @@
 import version
 
 
-base_choices = [
-    'ubuntu14_py27', 'ubuntu14_py34',
-    'ubuntu14_py35-pyenv', 'ubuntu14_py36-pyenv',
-    'ubuntu16_py27', 'ubuntu16_py35',
-    'centos6_py27-pyenv',
-    'centos7_py27', 'centos7_py34-pyenv']
+_base_choices = [
+    ('ubuntu14_py27', '2.7.6'),
+    ('ubuntu14_py34', '3.4.0'),
+    ('ubuntu14_py35-pyenv', '3.5.4'),
+    ('ubuntu14_py36-pyenv', '3.6.3'),
```

Comment:
How about using `get_python_version` in https://github.com/niboshi/chainer-test/blob/585df1196a116043ea163dbbe4417f2a8556a67f/docker.py#L195-L196

Reply:
Fixed.